dockstore-webservice/src/main/resources/migrations.1.16.0.xml

```diff
@@ -432,4 +432,21 @@
         <dropColumn tableName="workflowversion" columnName="aitopicprocessed">
         </dropColumn>
     </changeSet>
+    <changeSet author="svonworl" id="enableAutoDoiGeneration">
+        <sql dbms="postgresql">
+            <comment>Enable automatic DOIs for all entries</comment>
+            UPDATE apptool SET autogeneratedois = TRUE WHERE NOT archived;
+            UPDATE notebook SET autogeneratedois = TRUE WHERE NOT archived;
+            UPDATE service SET autogeneratedois = TRUE WHERE NOT archived;
+            UPDATE workflow SET autogeneratedois = TRUE WHERE NOT archived;
```

Comment:
Because we don't have any easy way to determine the provenance of if any existing `false` values (they could be the initial column value, or they could have been set to false in `.dockstore.yml`), this code potentially reverts any previous opt-outs.  But, an inspection of a recent database dump indicates that the only opt-outs are due to our own testing, so we're ok here, I think.